tests/page-objects/contacts/contacts.wdio.page.js

```diff
@@ -32,11 +48,47 @@ const getReportTaskFiltersText = async () => {
   return blah;
 };
 
+const addPlace = async (type, placeName , contactName ) => {
+  await (await resourceIcon(type.replace('_','-'))).click();
+  await (await newPrimaryContactButton()).click();
+  await (await newPrimaryContactName()).setValue(contactName);
```

Comment:
As a NIT, webdriver io has  `element.addValue()` function.  The difference being `setValue` clears the field first.